com.unity.netcode.gameobjects/package.json

```diff
@@ -6,7 +6,7 @@
     "unity": "2020.3",
     "dependencies": {
         "com.unity.nuget.mono-cecil": "1.10.1",
-        "com.unity.collections": "1.1.0",
-        "com.unity.transport": "1.0.0-pre.16"
+        "com.unity.collections": "1.2.3",
```

Comment:
why?

Reply:
It's the version transport 1.0.0 depends on and also the version that will be in the editor manifest. I don't think there's any strict need to update it here, but in effect 1.2.3 will be used anyway since it will be dragged along with transport.

Reply:
TL;DR: @ashwinimurt please revert collections package version change here :)

there are some other places within the NGO SDK we use stuff from the Collections package (`MessagingSystem`, `NetworkList` etc.).
even though NGO doesn't depend on a specific version of the Collection package and could get it through/from Transport package, that'd mean that potentially when Transport package gets updated with a newer version of the Collections package, it _might_ break NGO.
I think it's safer to keep NGO's Collections package dependency separate from UTP's Collection package dependency and update their versions on their own when needed, to keep things future-proof.

Reply:
Note that with the update to transport 1.0.0, NGO will use collections 1.2.3 no matter what. You can't have multiple versions of the same package installed AFAIK. So if transport brings in 1.2.3, that's what is going to be used by NGO even if its own dependency is at 1.1.0.

Reply:
> You can't have multiple versions of the same package installed AFAIK.

I'm not sure if that's true. Let's double-check?